src/cli/install.rs

```diff
@@ -15,18 +16,52 @@ pub struct Args {
     pub lock_file_usage: super::LockFileUsageArgs,
 
     #[arg(long, short)]
-    pub environment: Option<String>,
+    pub environments: Option<Vec<String>>,
```

Comment:
I guess this should be renamed to `environment` as it doesn't act like multiple input is possible:
```
pixi install --environments schema docs
```

Reply:
Ah that's right, back to environment you mean :)?